tdrs-backend/tdpservice/parsers/test/test_parse.py

```diff
@@ -887,10 +887,24 @@ def tanf_section3_file(stt_user, stt):
 
 
 @pytest.mark.django_db()
-def test_parse_tanf_section3_file(tanf_section3_file):
+def test_parse_tanf_section3_file(tanf_section3_file, dfs):
     """Test parsing TANF Section 3 submission."""
+    dfs.datafile = tanf_section3_file
+    dfs.save()
```

Comment:
nit: do you need to save the dfs?

Reply:
they were just holdovers from the old tests. i removed them and the tests still pass, thanks!